### DISCUSSION TITLE
C++ 20

### DISCUSSION THREAD

**Discussion by IHateYourCode** [texstudio-org/texstudio#1831] [Q&A] [answered]
As far as I was able to see, **TeXStudio** compiles
with `C++ 11`, however, is it planned to default to `C++ 20`
anytime soon or are there substantial problems with that?

**Comment by sunderme** [comment, marked as answer]
we are supporting „older“ linux distributions. It will be a while until they all support c++20.

**Reply by IHateYourCode** [reply]
Ahhh, right . .

**Comment by sunderme** [comment]
c++14 should work for now.
